Datadog/IntegrationUnitTests/RUM/SDKMetrics/RUMSessionEndedMetricIntegrationTests.swift

```diff
@@ -262,14 +263,6 @@ class RUMSessionEndedMetricIntegrationTests: XCTestCase {
     }
 }
 
-private extension RUMSessionEndedMetricIntegrationTests {
-    func rumMonitor(with sessionEndedSampleRate: SampleRate = .maxSampleRate) -> RUMMonitorProtocol {
-        let monitor = RUMMonitor.shared(in: core)
-        monitor.dd.scopes.dependencies.sessionEndedMetric.sampleRate = sessionEndedSampleRate
-        return monitor
-    }
-}
```

Comment:
cleanup/ This was hooking too much into implementation details, not good for integration testing. Replaced with passing the value from configuration:
```swift
rumConfig.sessionEndedMetricSampleRate = .maxSampleRate
```